frontend/src/lib-customizations/defaults/citizen/i18n/fi.tsx

```diff
@@ -1719,18 +1719,20 @@ export default {
           viranomainen on käsitellyt tiedot. Lomakkeen käsittelyn jälkeen voit
           päivittää tulotietojasi toimittamalla uuden lomakkeen.
         </P>
+
         <P>
           Kunnallisen varhaiskasvatuksen asiakasmaksut määräytyvät
           prosenttiosuutena perheen bruttotuloista. Maksut vaihtelevat perheen
-          koon ja tulojen sekä varhaiskasvatusajan mukaan. Tarkista{' '}
+          koon ja tulojen sekä varhaiskasvatusajan mukaan. Tarkista
+          asiakastiedotteen taulukosta (jonka löydät{' '}
```

Comment:
I would prefer `<a href="...">asiakastiedotteen taulukosta</a>` but maybe the spec was to have it like this in parentheses?

Reply:
I think this is because not everyone is trusted to figure out blue text being a link